docker/cloudsim_bridge/run_bridge.bash

```diff
@@ -21,4 +21,4 @@ export ROSCONSOLE_STDOUT_LINE_BUFFERED=1
 # atop process monitoring
 atop -R -w /home/developer/.ros/atop_log &
 
-ign launch cloudsim_bridge.ign -v 4 $@
+ign launch cloudsim_bridge.ign -v 4 $@ | tee /home/developer/.ros/bridge.log
```

Comment:
We should pipe `stderr` as well.

```suggestion
ign launch cloudsim_bridge.ign -v 4 $@ |& tee /home/developer/.ros/bridge.log
```

Reply:
Do we want to log `stderr` in the `stdout` too?

Reply:
Yes, we will lose that information if we don't.

Reply:
We can copy whatever comes from stderr into stdout and then logging that into a file by doing this:
`ign launch ... 2>&1 | tee`